Core/Sources/Model/LocationUpdate.swift

```diff
@@ -3,15 +3,15 @@ import CoreLocation
 /**
  Enumeration used to determine enhanced location type.
  */
-public enum LocationUpdateType: String, Codable {
+enum LocationUpdateType: String, Codable {
```

Comment:
It should remain `public` just like `EnhancedLocationUpdate` :wink:

Reply:
Cool, reverted it 👍